src/classic.c

```diff
@@ -10,5 +10,6 @@ bool is_running_on_classic_distribution()
 	    || access("/var/lib/dpkg/status", F_OK) == 0
 	    || access("/var/lib/pacman", F_OK) == 0
 	    || access("/var/lib/portage", F_OK) == 0
-	    || access("/var/lib/rpm", F_OK) == 0;
+	    || access("/var/lib/rpm", F_OK) == 0
+	    || access("/sbin/procd", F_OK) == 0;
```

Comment:
Shouldn't this be detecting the opkg database path rather than `/sbin/procd`? It doesn't seem to make sense to detect an init system for detecting openwrt as a classic distro.